.gitignore

```diff
@@ -37,3 +37,7 @@ terraform.rc
 .terraform.lock.hcl
 
 build
+target
+bin
+obj
+.DS_Store
```

Comment:
You could also add `.DS_Store` to your global git ignore file at `~/.gitignore`. Here's what's in mine
```
storage
compile-cache
dev
.npm
.node-gyp
coverage.out

node_modules/

.bundle

.vim/.netrwhist

# Suggested from atom's gitignore
# https://github.com/atom/atom/blob/17a31e3a3729070768f31bbce7ce9bcc09f5a2b8/dot-atom/.gitignore
.atom/blob-store
.atom/compile-cache
.atom/dev
.atom/storage
.atom/.apm
.atom/.node-gyp
.atom/.npm
.atom/.atom-socket-secret-*

# https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
!.vscode/*.code-snippets

# Local History for Visual Studio Code
.history/

# Built Visual Studio Code Extensions
*.vsix

.tmux/resurrect/

# kernprof output
*.lprof

# copilot
.vim/pack/github/start/copilot.vim
```

Reply:
Huh, funny that I don't have .DS_Store in there....

Reply:
TIL you can configure global `.gitignore` files!